src/invoice/printable-invoice/printable-invoice-header.component.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { type PatientDetails } from '../../types';
 import { type SessionLocation, useConfig, interpolateUrl } from '@openmrs/esm-framework';
+import { age, formatPartialDate } from '@openmrs/esm-utils';
```

Comment:
```suggestion
import { type SessionLocation, useConfig, interpolateUrl, age, formatPartialDate } from '@openmrs/esm-framework';
```

All API functions from Core should be imported from `@openmrs/esm-framework` instead of its constituent libraries.